apps/judicial-system/backend/src/app/modules/police/police.controller.ts

```diff
@@ -76,28 +69,6 @@ export class PoliceController {
     return this.policeService.getAllPoliceCaseFiles(theCase.id, user)
   }
 
-  @RolesRules(
-    prosecutorRule,
-    prosecutorRepresentativeRule,
-    districtCourtJudgeRule,
-    districtCourtAssistantRule,
-    districtCourtRegistrarRule,
-  )
-  @Get('subpoenaStatus/:subpoenaId')
-  @ApiOkResponse({
-    type: Subpoena,
-    description: 'Gets subpoena status',
-  })
-  getSubpoenaStatus(
```

Comment:
Subpoena refreshes have been moved to the subpoena module and is no longer done through separate backend endpoints.